Modules/Core/SpatialObjects/test/itkImageMaskSpatialObjectGTest.cxx

```diff
@@ -153,13 +153,13 @@ TEST(ImageMaskSpatialObject, AxisAlignedBoundingBoxIsEmptyWhenAllPixelsAreZero)
 {
   // Test 2D images:
   Expect_AxisAlignedBoundingBoxRegion_is_empty_when_all_pixel_values_are_zero<double>(
-    itk::ImageRegion<2>{ itk::Index<2>{}, itk::Size<2>::Filled(1) });
+    itk::ImageRegion<2>{ itk::Index<2>{{0, 0}}, itk::Size<2>::Filled(1) });
   Expect_AxisAlignedBoundingBoxRegion_is_empty_when_all_pixel_values_are_zero<double>(
     itk::ImageRegion<2>{ itk::Index<2>{{-1, -2}}, itk::Size<2>{{3, 4}} });
 
   // Test 3D images:
   Expect_AxisAlignedBoundingBoxRegion_is_empty_when_all_pixel_values_are_zero<unsigned char>(
-    itk::ImageRegion<3>{ itk::Index<3>{}, itk::Size<3>::Filled(1) });
+    itk::ImageRegion<3>{ itk::Index<3>{{0, 0}}, itk::Size<3>::Filled(1) });
```

Comment:
In order to please the old compiler, please consider using parentheses: `itk::Index<3>()`.
Note that `itk::Index<3>{{0, 0}}` might trigger a warning anyway, because of the number of zero's!

Reply:
Great catch @N-Dekker -- fixed!